README.md

```diff
@@ -38,16 +39,20 @@ The [check-controller-build](.github/actions/check-controller-build) action is u
 
 #### get-benchmarks
```

Comment:
get-benchmarks (deprecated)

Reply:
just add a more clore note about deprecation at the top...
```
# foobar
Note: This option is **DEPRECATED**
```

Reply:
How is it now?